src/components/LoginForm.js

```diff
@@ -4,19 +4,49 @@ import SignupButton from "../components/SignupButton";
 import TextFieldContainer from "../components/TextFieldContainer";
 import styled from "styled-components";
 
-const LoginFormStyled = styled.div`
-  display: flex;
+const LoginFormStyled = styled.form`
   width: 100vw;
   height: 100vh;
 `;
 
+const DivEmail = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 60px;
+`;
+
+const DivSignupButton = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
 export default function LoginForm() {
   return (
     <>
-      <LoginFormStyled />
       <Globalstyles />
-      <SignupButton />
-      <TextFieldContainer />
+      <LoginFormStyled>
+        <DivEmail>
+          <TextFieldContainer
+            input
+            type="email"
+            placeholder="Enter Email"
+            name="email"
+            required
+          ></TextFieldContainer>
+          <TextFieldContainer
+            input
+            type="password"
+            placeholder="Enter Password"
+            name="psw"
+            required
+          ></TextFieldContainer>
+        </DivEmail>
+        <DivSignupButton>
+          <SignupButton>Login</SignupButton>
+          <SignupButton>Sign Up</SignupButton>
+        </DivSignupButton>
+      </LoginFormStyled>
```

Comment:
The structure could use some work, `DivEmail` contains the input fields for the login right? You could name it `LoginWrapper` for example. `DivSignupButton` contains both login and signup buttons, so you should name it appropriately as well, maybe `ButtonWrapper`?

Also it is better to not use `100vw` and `100vh` in a component that could be used somewehre else than a full screen login page. If you give `height` and `width` both 100%, you can have a parent container setting the size a bit more dynamically.